code/game/objects/explosion.dm

```diff
@@ -112,10 +114,11 @@ proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impa
 			var/throw_dir = get_dir(epicenter,T)
 			for(var/obj/item/I in T)
 				spawn(0) //Simultaneously not one at a time

```

Comment:
throw_at uses sleep in it, so ya, you need the spawn, other wise things would wait until the last item reached its destination before moving on to the next